README.md

```diff
@@ -2,7 +2,17 @@
 
 [![npm](https://img.shields.io/npm/v/iab-vast-model.svg)](https://www.npmjs.com/package/iab-vast-model) [![Dependencies](https://img.shields.io/david/zentrick/iab-vast-model.svg)](https://david-dm.org/zentrick/iab-vast-model) [![Build Status](https://img.shields.io/circleci/project/github/zentrick/iab-vast-model/master.svg)](https://circleci.com/gh/zentrick/iab-vast-model) [![Coverage Status](https://img.shields.io/coveralls/zentrick/iab-vast-model/master.svg)](https://coveralls.io/r/zentrick/iab-vast-model) [![JavaScript Standard Style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
 
-Base classes for IAB VAST handling. Supports the full IAB VAST 3.0 specification.
+Base classes for IAB VAST handling. Supports the full IAB VAST 2.0, 3.0 and 4.0
+specifications.
+
+Note that the `<Creatives>`, `<CreativeExtensions>`, `<Extensions>` and
+`<Verifications>` are not directly recorded since they have no attributes. For
+instance, the list of `<creative>` are recorded directly by `<Inline>` and
+`<Wrapper>`.
+
+Please check directory `docs/dump/`. It contains VAST files along with JSON
```

Comment:
While I think examples are definitely a good idea, I'm not sure the current contents of `dump` are helpful. They contain the private, underscore-prefixed properties rather than the getters, which don't always have the same name.

However, you can find helper code that translates a class instance to a pure JSON object based on its getters in the ad-tester-react repository. It's being used to visualize a VAST tree on tester.zentrick.io already. We're going to open-source that once this PR has been merged so it's fine if you copy that part over already.

Reply:
Also, the examples probably shouldn't be the first option. We're autogenerating docs in HTML so let's definitely make it clear that those are also available rather than pointing people to JSON dumps.

Reply:
I have removed the `dump/` directory.